android/app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,7 @@
         <meta-data android:name="commitHash" android:value="${commitHash}"/>
         <activity
                 android:name=".MainActivity"
+                android:exported="true"
```

Comment:
What does this do?

Reply:
this states if an activity can be launched from outside the app 
https://developer.android.com/google/play/requirements/target-sdk#:~:text=Intent%20filters%3A%20If%20your
https://developer.android.com/guide/topics/manifest/activity-element#exported